modules/nw-networkpolicy-about.adoc

```diff
@@ -64,7 +64,8 @@ spec:
 If the Ingress Controller is configured with `endpointPublishingStrategy: HostNetwork`, then the Ingress Controller pod runs on the host network.
 When running on the host network, the traffic from the Ingress Controller is assigned the `netid:0` Virtual Network ID (VNID).
 The `netid` for the namespace that is associated with the Ingress Operator is different, so the `matchLabel` in the `allow-from-openshift-ingress` network policy does not match traffic from the `default` Ingress Controller.
-Because the `default` namespace is assigned the `netid:0` VNID, you can allow traffic from the `default` Ingress Controller by labeling your `default` namespace with `network.openshift.io/policy-group: ingress`.
+Because the `default` namespace is assigned the `netid:0` VNID, you can allow traffic from the `default` Ingress Controller by labeling your `default` namespace with `network.openshift.io/policy-group: ingress` if you use OpenShift SDN.
+For the OVN-Kubernetes network provider plug-in, there is no supported configuration to allow traffic from Ingress Controller pods on the host network and use a network policy to deny traffic from all other sources.
```

Comment:
@mikemckiernan sorry for late reply. hmm,  seems I did not understand this totally. 
@astoycos hi, I still confused this, as https://bugzilla.redhat.com/show_bug.cgi?id=1927841#c10 said.  'Hostnetwork' pod can access any `pod-networking` even there is isolation of project.  and it's expected. However the bug is talking about app cannot be accessed from outside( means router pod cannot be accessed the app pod ip). is this same issue?

Reply:
@zhaozhanqi, I think that this docs update is only to specify that the workaround that works for OpenShift SDN does not work for OVN-Kubernetes at this time. Is that accurate?

Thanks!

Reply:
yes, I just understand the logic here. sorry I made a mistake before.